Unity-Technologies/com.unity.mobile.logcat
# Fix Addr2Line functionality + Improvements

* Implemented custom way of calling addr2line, that way we depend less on Unity. (Unity actually broke addr2line in 2019.3)
* Add functionality Filter by process id
* Correctly open Terminal on macOS Catalina
https://github.com/Unity-Technologies/com.unity.mobile.logcat/issues/69
https://github.com/Unity-Technologies/com.unity.mobile.logcat/issues/60
https://github.com/Unity-Technologies/com.unity.mobile.logcat/issues/59
https://github.com/Unity-Technologies/com.unity.mobile.logcat/issues/57

Things to test:
* Check if stacktrace resolving works correctly on Mac and Windows, simply open stacktrace window, set correct symbol path, and click Resolve Stacktraces
* Check if Filter by process id works
* Check if Terminal opens up on macOS Catalina https://fogbugz.unity3d.com/f/cases/1213731/
* https://fogbugz.unity3d.com/f/cases/1213954/ Logcat scripts shouldn't be auto updated
